service/job/job_controller.py

```diff
@@ -287,6 +294,7 @@ def job_create(job_type):
     job_db.add_job(job_id, user, job_type, task_ids, params)
 
     body = jsonify({
+        'success': True,
```

Comment:
Wozu brauchen wir success wenn es eh immer True ist ?

Reply:
Wenn er (weiter oben) in einen Fehlerfall läuft, wird eine "ApiError"-Exception generiert, die da false liefert.
Siehe: service/errors.py